server/index.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+
+const { PORT } = process.env; // setup PORT to be used
+const app = express(); // calling an instance of express
+
+app.get('/', (request, response) => {
+    try{
+        response.status(200).send('Hello Books');
+    } catch (error){
```

Comment:
What I mean by error handlers are handlers that catch routes that do not exist and return a 404 or 500 depending on the error.

Reply:
Done, treated accordingly